src/collection-facets.ts

```diff
@@ -585,9 +585,10 @@ export class CollectionFacets extends LitElement {
         padding: 0;
         background: inherit;
         border: 0;
-        color: blue;
+        color: #4b64ff;
```

Comment:
can we get a var? `--collectionBrowserLinkColor` or something so that all the other anchor colored links get the same treatment?